src/EprPrnIntegration.Api/HostBuilderConfiguration.cs

```diff
@@ -47,6 +51,7 @@ private static void ConfigureServices(IConfiguration configuration, IServiceColl
         services.AddScoped<IServiceBusProvider, ServiceBusProvider>();
         services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
         services.AddSingleton<IConfigurationService, ConfigurationService>();
+        services.AddSingleton<IEmailService, EmailService>();
```

Comment:
Shouldn't it be registered as scoped?

Reply:
Email services are Singleton.
We use Scoped if EmailService depends on other scoped services or needs to manage per-request state like DB contex.